sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -251,9 +251,10 @@ object ScalaJSPlugin extends AutoPlugin {
         "Prints the content of a .sjsir file in human readable form.",
         CTask)
 
-    val scalaJSConfigurationLibs = TaskKey[Seq[ResolvedJSDependency]](
-        "scalaJSConfigurationLibs",
-        "List of JS libraries used as project configuration.", CTask)
+    val scalaJSJSFiles = TaskKey[Seq[VirtualJSFile]](
```

Comment:
It's definitely not an ideal name. I like `jsExecutionFiles`. Or maybe even `jsEnvFiles` as they are mainly supposed to be used with a JS env.